README.md

```diff
@@ -1,159 +1,25 @@
-
-```sh
-filetype plugin on
-```
-
-## Usage
-
-### Documentation
-
-Please see the vim help system for full documentation of all options: `:help nerdcommenter`
-
-### Settings
-
-Several settings can be added to your vimrc to change the default behavior. Some examples:
-
-```vim
-" Add spaces after comment delimiters by default
-let g:NERDSpaceDelims = 1
-
-" Use compact syntax for prettified multi-line comments
-let g:NERDCompactSexyComs = 1
-
-" Align line-wise comment delimiters flush left instead of following code indentation
-let g:NERDDefaultAlign = 'left'
-
-" Set a language to use its alternate delimiters by default
-let g:NERDAltDelims_java = 1
-
-" Add your own custom formats or override the defaults
-let g:NERDCustomDelimiters = { 'c': { 'left': '/**','right': '*/' } }
-
-" Allow commenting and inverting empty lines (useful when commenting a region)
-let g:NERDCommentEmptyLines = 1
-
-" Enable trimming of trailing whitespace when uncommenting
-let g:NERDTrimTrailingWhitespace = 1
-
-" Enable NERDCommenterToggle to check all selected lines is commented or not 
-let g:NERDToggleCheckAllLines = 1
-```
-
-### Default mappings
-
-The following key mappings are provided by default (there is also a menu provided that contains menu items corresponding to all the below mappings):
-
-Most of the following mappings are for normal/visual mode only. The **|NERDComInsertComment|** mapping is for insert mode only.
-
-  * `[count]<leader>cc` **|NERDComComment|**
-
-    Comment out the current line or text selected in visual mode.
-
-  * `[count]<leader>cn` **|NERDComNestedComment|**
-
-    Same as <leader>cc but forces nesting.
-
-  * `[count]<leader>c<space>` **|NERDComToggleComment|**
-
-    Toggles the comment state of the selected line(s). If the topmost selected line is commented, all selected lines are uncommented and vice versa.
-
-  * `[count]<leader>cm` **|NERDComMinimalComment|**
-
-    Comments the given lines using only one set of multipart delimiters.
-
-  * `[count]<leader>ci` **|NERDComInvertComment|**
-
-    Toggles the comment state of the selected line(s) individually.
-
-  * `[count]<leader>cs` **|NERDComSexyComment|**
-
-    Comments out the selected lines with a pretty block formatted layout.
-
-  * `[count]<leader>cy` **|NERDComYankComment|**
-
-    Same as <leader>cc except that the commented line(s) are yanked first.
-
-  * `<leader>c$` **|NERDComEOLComment|**
-
-    Comments the current line from the cursor to the end of line.
-
-  * `<leader>cA` **|NERDComAppendComment|**
-
-    Adds comment delimiters to the end of line and goes into insert mode between them.
-
-  * **|NERDComInsertComment|**
-
-    Adds comment delimiters at the current cursor position and inserts between. Disabled by default.
-
-  * `<leader>ca` **|NERDComAltDelim|**
-
-    Switches to the alternative set of delimiters.
+For initial instructions, check out the original script at
```

Comment:
In order to consider this PR for upstreaming can we revert these README changes and anything else that doesn't belong being merged upstream? If you want to switch to doing this on a branch instead of your fork's master that's fine.

Reply:
Ok, I'll try that.  Any branch name suggestions?

Reply:
I'd just call it `autoload`. Then I'd rebase it on upstream/master and drop the any unrelated commits and open a new PR. I'll close this one when you do since it's against your `master`.

Reply:
Branch created.

Reply:
Great! Can you [open it as a PR too](https://github.com/scrooloose/nerdcommenter/compare/master...Houl:autoload?expand=1? That way it will show up as being your contribution.